src/test/java/com/bulkbuy/enterprise/EnterpriseApplicationTests.java

```diff
@@ -146,7 +141,7 @@ void confirmFindByOrderId() throws Exception {
         Order returnedOrder2 = orderService.create(order2);
 
         Order orderLookup = orderService.findByOrderId(orderId2);
-        assertEquals(order2, orderLookup);
+        assertEquals(returnedOrder2, orderLookup);
```

Comment:
I noticed you were getting returnedOrder2 but not using it anywhere, I thought it would be effective to implement it in the test.